code/game/objects/items/binoculars.dm

```diff
@@ -168,7 +168,7 @@
 		to_chat(user, "<span class='warning'>You can't focus properly through \the [src] while looking through something else.</span>")
 		return
 
-	if(!can_see(user, A, 25))
+	if(!(A in view(user,25)))
```

Comment:
This is terribad and should use a proper view check instead of this hellishness
If I cant think of wany when I have time later I'll be ok with removing it